bridge/pier/common.go

```diff
@@ -85,6 +87,32 @@ func isProposer(cliCtx cliContext.CLIContext) bool {
 	return false
 }
 
+// IsStateSyncer returns if current user is state syncer or not
+func IsStateSyncer(cliCtx cliContext.CLIContext) bool {
+	var stateSyncerList []types.Validator
+	isStateSyncer := false
+
+	result, err := FetchFromAPI(cliCtx, GetHeimdallServerEndpoint(fmt.Sprintf(StateSyncerURL)))
```

Comment:
What if all three validators are offline?

Reply:
That would be the case if we go with backup model also.
Ideally, Instead of 3 validators We should increase the number to 1/3rd of currentvalidatorset count. It results in more number of transactions per sync.

Let me know if there is any better idea.

Reply:
Yeah 1/3rd of current validator set would be better here and in bor as backup.